### PR TITLE
Add capeless/warrant.

### PR DESCRIPTION
**Pull request by armicron**
**Why is this awesome?**

[**Warrant**](https://github.com/capless/warrant) has `AWSSRP` class that is used to perform SRP (Secure Remote Password protocol) authentication. This is a preferred method of user authentication with AWS Cognito. The process involves a series of authentication challenges and responses, which if successful, results in a final response that contains ID, access and refresh tokens.

Python SDK boto3 lacks the support of SRP.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -521,6 +521,7 @@ AWS Repos:
 Community Repos:
 
 * [Contribute](https://github.com/donnemartin/awesome-aws/blob/master/CONTRIBUTING.md)
+* [warrant](https://github.com/capless/warrant) - Python library for using AWS Cognito.
```
Comment (donnemartin) [inline README.md]: Can you make the displayed text capless/warrant for consistency?

File: README.md [hunk 2]
```diff
@@ -521,6 +521,7 @@ AWS Repos:
 Community Repos:
 
 * [Contribute](https://github.com/donnemartin/awesome-aws/blob/master/CONTRIBUTING.md)
```
Comment (donnemartin) [inline README.md]: We could remove this now that there's an item in this section, also for consistency.

File: README.md (hunk 1 above)
Comment (armicron) [reply README.md]: Yes, of course. I will make new PR, as I deleted a fork and can't make any change. Thank you for review.

Comment (armicron) [reply README.md]: new PR https://github.com/donnemartin/awesome-aws/pull/112

**Pull request closed without merging**